gym/envs/mujoco/mujoco_env.py

```diff
@@ -132,7 +129,7 @@ def do_simulation(self, ctrl, n_frames):
 
     def render(
         self,
-        mode="human",
+        mode,
```

Comment:
I think many codebases assume that the default value is mode="human" so it might be better to keep it this way.